src/tests/nativeaot/SmokeTests/ComWrappers/ComWrappersNative.def

```diff
@@ -0,0 +1,6 @@
+EXPORTS
+	BuildComPointer
```

Comment:
Why do we need these explicit .def exports?

Reply:
Otherwise the mangled names are exported. This is an unfortunate x86 quirk. MinGW linker has the `—kill-at` switch to get rid of that but MSVC linker doesn’t. Cmake can generate .def files that but it exports everything and not just the `__declspec(dllexport)` methods. I didn’t find a way to so this automatically with existing build tools.

Reply:
DllImport is expected to probe for the mangled name on x86: https://github.com/dotnet/runtime/blob/da781b3aab1bc30793812bced4a6b64d2df31a9f/src/coreclr/vm/method.cpp#L3449-L3463 . Would it be better to implement this probing for native AOT instead?

Reply:
Interesting. I'll check how much effort is it to get this working.